backend/src/main/kotlin/com/tngtech/apicenter/backend/connector/rest/controller/ServiceController.kt

```diff
@@ -83,4 +83,7 @@ class ServiceController @Autowired constructor(
     @GetMapping("/search/{searchString}")
     fun searchForServices(@PathVariable searchString: String): List<ServiceDto> =
         serviceHandler.search(searchString).map { spec -> serviceDtoMapper.fromDomain(spec) }
+
+    @GetMapping("/search/")
+    fun searchWithBlank(): List<ServiceDto> = listOf()
```

Comment:
Interesting that this isn't caught by the previous method.

Reply:
It makes sense to me when thinking of a rest API that returns a collection of all entries for `/entries` and a specific entry for `/entries/{uuid}`, for example. I believe ApiCenter implements something similar. This workaround makes me uneasy because it is logically an exception case of the GetMapping above, and the code doesn't reflect that. But I'm not sure if it's clearer to do something like this https://stackoverflow.com/a/34670841